website/handlers/router.ts

```diff
@@ -153,7 +154,12 @@ export default function RoutesSelection(
 ): Handler {
   return async (req: Request, connInfo: ConnInfo): Promise<Response> => {
     // TODO: (@tlgimenes) Remove routing from request cycle
```

Comment:
Shouldn't the TODO be moved to the line below?

Reply:
It is an overall comment about the route function. Maybe it should be one line above, but we don't have a defined comment style on how should we do it.